### PR TITLE
docs and parameterization

### PR DESCRIPTION
**Pull request by compwron**
Please let me know if I broke any functionality from your end.

### CODE REVIEW

**Comment by samfoo** [comment]
Hmmm... I think at this point using the environment for configuration has gotten a bit out of control. Seems like a config file makes more sense at this point.

**Comment by samfoo** [comment]
readme demands more humour.  (consider this a blocker ;-)

File: jcvd [hunk 1]
```diff
@@ -81,40 +80,19 @@ def email(badguy, subject, body)
...
-      :user_name => ENV["TWUSER"],
-      :password => ENV["TWPASS"],
-      :authentication => :plain,
-      :domain => "HELO",
-    },
-    :subject => subject.strip, :body => body)
-  end
-end
-
-def karate
```
Comment (quad) [inline jcvd:97]: What the hell? Removing the `--break` option?!

File: tec [hunk 2]
```diff
@@ -30,25 +31,21 @@ end
 
 deadline = due_date.strftime("%d/%m/%Y")
 
-user = ENV["PUSER"] 
-pass = ENV["PPASS"] 
-$stderr.puts "They killed his timesheet ten years ago... on the week ending #{deadline}..."
```
Comment (quad) [inline tec:35]: Do you have a soul?

Comment (compwron) [inline tec:35]: Not sure. The man page does not mention one, though. :)

**Pull request closed without merging**